src/Model/Consignment/AbstractConsignment.php

```diff
@@ -1355,6 +1361,31 @@ public function setAgeCheck(bool $ageCheck): self
         return $this;
     }
 
+    /**
+     * @return bool
+     */
+    public function hasCooledDelivery(): bool
```

Comment:
Nu kan in AbstractShipmentOptionsAdapter.php een null teruggeven. Dit terwijl je eigenlijk met has altijd een bool verwacht. Dan kan je in de consignment zelf nog wel checken op null als je die field aanspreek (toch?).

Reply:
done